File: data/Cgi/TestQuery.php
<?php
// Get the values of query parameters
$param1 = $_GET['param1'];
$param2 = $_GET['param2'];

if (isset($param1) && isset($param2)) {
    echo "Value of param1: $param1<br>";
    echo "Value of param2: $param2<br>";
} else {
    echo "One or both parameters are missing in the query string.";
}
?>

File: Cgi/TestQuery.py
#!/usr/bin/python
import cgi

form = cgi.FieldStorage()

param1 = form.getvalue('param1')
param2 = form.getvalue('param2')

if param1 and param2:
    print("Value of param1: " + param1 + "<br>")
    print("Value of param2: " + param2 + "<br>")
else:
    print("One or both parameters are missing in the query string.")
